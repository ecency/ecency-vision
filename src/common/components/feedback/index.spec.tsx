import React from "react";

import renderer from "react-test-renderer";

import Feedback from "./index";

describe("Feedback component", () => {
  it("renders correctly", () => {
    const props = {
      activeUser: null
    };
    const tree = renderer.create(<Feedback {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
