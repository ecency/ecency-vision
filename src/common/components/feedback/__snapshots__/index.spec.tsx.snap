// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Feedback component renders correctly 1`] = `
<div
  className="feedback-container"
/>
`;
